client/me/help/main.jsx

```diff
@@ -262,11 +266,7 @@ class Help extends React.PureComponent {
 	};
 
 	trackSupportSessionButtonClick = () => {
-		//@TODO: What information do we want to track here?
-		const { isBusinessPlanUser } = this.props;
-		recordTracksEvent( 'calypso_help_support_session_card_click', {
```

Comment:
Can we leave the Tracks event, just not worry about the plan info?

Reply:
Oh, silly me. I thought you meant remove the tracking entirely. I re-added it on click. 😅